index.html

```diff
@@ -41,8 +41,170 @@ <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</sp
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="work-container">
+            <div class="img-placer">
+                <img src="images/im1.png" alt="">
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="work-container">
+            <div class="img-placer">
+                <img src="images/im1.png" alt="">
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="work-container">
+            <div class="img-placer">
+                <img src="images/im1.png" alt="">
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="work-container">
+            <div class="img-placer">
+                <img src="images/im1.png" alt="">
+            </div>
+            <div class="txt-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="">Ruby on rails</a></li>
+                    <li><a href="">css</a></li>
+                    <li><a href="">JavaScript</a></li>
+                    <li><a href="">html</a></li>
+                </ul>
+                <button class="see-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+    </section>
+    <section class="abt">
```

Comment:
Great job @Tekhlay 🤝

- [ ] Kindly align the contents of the `About Me Section` with `display: flex`.